src/Export/Search/AbstractProductSearcher.php

```diff
@@ -8,6 +8,7 @@
 use FINDOLOGIC\Shopware6Common\Export\Config\PluginConfig;
 use FINDOLOGIC\Shopware6Common\Export\Utils\Utils;
 use InvalidArgumentException;
+use Shopware\Core\Content\Product\Aggregate\ProductVisibility\ProductVisibilityDefinition;
```

Comment:
There is no Shopware within the library, this has to be some class from `Vin\ShopwareSdk`

Reply:
Aren't those exactly the ones you are using, or did you add them for the screenshot?

But regarding what exists and what not, we can't use Shopware classes within shopware6-common.

Reply:
I just added them for the screenshot, they didn't exist before.
I understand that we cannot use Shopware's classes within shopware6-common.

Reply:
If there is no constant, we can either define it ourselves in the class, or just use a hardcoded value

Reply:
What do you suggest regarding this matter, or how should we add these constants? Do you perhaps have access to the git repository for `vin-sw` where the `Vin\ShopwareSdk` classes are located (I'm just assuming that we could add them there)?

Reply:
Can we add the constant to the `ProductVisibilityDefinition` class located within the `Vin\ShopwareSdk`, or did you mean to add it to the `AbstractProductSearcher` class?

Reply:
You can ask for it here: https://github.com/vienthuong/shopware-php-sdk

But for now, I would just put it within the AbstractProductSearcher